flink-connectors/flink-connector-kinesis/pom.xml

```diff
@@ -36,6 +36,8 @@ under the License.
 		<aws.sdk.version>1.10.71</aws.sdk.version>
 		<aws.kinesis-kcl.version>1.6.2</aws.kinesis-kcl.version>
 		<aws.kinesis-kpl.version>0.10.2</aws.kinesis-kpl.version>
+		<apache.httpclient.version>4.3.6</apache.httpclient.version>
+		<apache.httpcore.version>4.3.3</apache.httpcore.version>
```

Comment:
yes.
Here's the dependency tree:

aws-java-sdk-kinesis 1.10.71
aws-java-sdk-core 1.10.71
httpclient 4.3.6
httpcore 4.3.3